libavfilter/dnn/dnn_backend_tf.c

```diff
@@ -546,11 +545,13 @@ static DNNReturnType add_maximum_layer(TFModel *tf_model, TF_Operation **cur_op,
     *y = params->val.y;
     TF_SetAttrTensor(op_desc, "value", tensor, tf_model->status);
     if (TF_GetCode(tf_model->status) != TF_OK){
+        TF_DeleteTensor(tensor);
         av_log(ctx, AV_LOG_ERROR, "Failed to set value for maximum/y of layer %d", layer);
         return DNN_ERROR;
     }
     op = TF_FinishOperation(op_desc, tf_model->status);
     if (TF_GetCode(tf_model->status) != TF_OK){
+        TF_DeleteTensor(tensor);
```

Comment:
line 569? I'm not sure, need a try in gdb

Reply:
maybe TF_DeleteGraph is enough since it has been added into graph?

Reply:
I tried it in gdb, it looks DeleteGraph will not DeleteTensor at the same time (deleting one TF object twice will report error, while DeleteTensor after DeleteGraph will not report error)

Reply:
ok, and please check line 569

Reply:
both line 521 & 569 are added, and i have checked, no more tensor mem free is missing